src/protagonist/Engine.Tests/Ingest/IngestExecutorTests.cs

```diff
@@ -18,7 +18,8 @@ public IngestExecutorTests()
     {
         workerBuilder = A.Fake<IWorkerBuilder>();
         repo = A.Fake<IEngineAssetRepository>();
-        sut = new IngestExecutor(workerBuilder, repo, new NullLogger<IngestExecutor>());
+        var assetSizeCheck = A.Fake<IAssetIngestorSizeCheck>();
+        sut = new IngestExecutor(workerBuilder, repo, assetSizeCheck ,new NullLogger<IngestExecutor>());

```

Comment:
Should there be a test that uses `assetSizeCheck`?

Tiny thing but spacing around comma when passing it is off : `assetSizeCheck ,new Null...`